Examples/UICatalog/Scenarios/ConfigurationEditor.cs

```diff
@@ -3,7 +3,6 @@
 using System.IO;
 using System.Linq;
 using System.Reflection;
-using Terminal.Gui;
```

Comment:
Can you ensure the `Dialog` cancel button does not close this scenario, please. Thanks.